<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>overlay</class>
 <widget class="QMainWindow" name="overlay">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>438</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>598</width>
    <height>438</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>598</width>
    <height>438</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Overlay</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>598</width>
     <height>418</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>601</width>
     <height>418</height>
    </size>
   </property>
   <widget class="QPushButton" name="convert_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>390</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="output_group">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>601</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="output_browse">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QComboBox" name="output_dropdown">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>491</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QCheckBox" name="output_folder_checkbox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Output To Original Folder</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="extension_inp">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>0</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>mov</string>
     </property>
    </widget>
    <widget class="QLabel" name="extension_label">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>-2</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Extension:</string>
     </property>
    </widget>
    <widget class="QLabel" name="original_resolution_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>of original resolution</string>
     </property>
    </widget>
    <widget class="QLabel" name="resolution_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Output Resolution:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="resolution_dropdown">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="information_group">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Information</string>
    </property>
    <widget class="QLabel" name="project_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Project</string>
     </property>
    </widget>
    <widget class="QLabel" name="author_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="author_inp">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="task_label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Task</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="project_inp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="version_inp">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="version_label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="task_inp">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="file_list_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>File List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>More Tool</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::MenuButtonPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>581</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="preser_group">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="preset_dropdown">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="preset_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Presets</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="update_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Update: 21/08/2019</string>
    </property>
   </widget>
   <widget class="QListWidget" name="file_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>571</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="input_browse">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
